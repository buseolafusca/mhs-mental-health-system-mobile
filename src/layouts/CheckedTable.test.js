import React from 'react'
import ReactDOM from 'react-dom'
import SelectableTable from '../components/CheckedTable/CheckedTable'
import { shallow } from 'enzyme'

it('renders without crashing', () => {
  
  const component = shallow(<SelectableTable />)
  expect(component).toMatchSnapshot()

})
