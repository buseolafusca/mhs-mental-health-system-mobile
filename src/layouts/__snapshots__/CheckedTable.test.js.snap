// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<Table
  bordered={true}
  columns={
    Array [
      Object {
        "dataIndex": "checked",
        "render": [Function],
        "title": "Select service",
        "width": 80,
      },
      Object {
        "dataIndex": "serviceName",
        "title": "Service Name",
      },
    ]
  }
  data={Array []}
  hoverable={true}
  justified={true}
  loading={false}
  maxHeight={0}
  onRowClick={[Function]}
  rowClassName={[Function]}
  rowKey="id"
  useFixedHeader={false}
/>
`;
